diff --git a/core/lib/Drupal/Core/Entity/ContentEntityBase.php b/core/lib/Drupal/Core/Entity/ContentEntityBase.php
index 1b672e62021..10b6e0073a4 100644
--- a/core/lib/Drupal/Core/Entity/ContentEntityBase.php
+++ b/core/lib/Drupal/Core/Entity/ContentEntityBase.php
@@ -978,6 +978,9 @@ protected function initializeTranslation($langcode) {
    * {@inheritdoc}
    */
   public function hasTranslation($langcode) {
+    if ($langcode === NULL) {
+      $langcode = '';
+    }
     if ($langcode == $this->defaultLangcode) {
       $langcode = LanguageInterface::LANGCODE_DEFAULT;
     }
